index.d.ts

```diff
@@ -0,0 +1,2 @@
+export default function (image: string | HTMLImageElement): Promise<HTMLImageElement>;
+export default function (images: string[] | HTMLImageElement[]): Promise<HTMLImageElement[]>;
```

Comment:
Can this benefit from the info in this thread? https://github.com/Microsoft/TypeScript/issues/424#issuecomment-173902626

Perhaps `ArrayLike< HTMLImageElement >`?